e2e_tests/tests/experiment/experiment.py

```diff
@@ -914,3 +914,22 @@ def s3_checkpoint_config_no_creds() -> Dict[str, str]:
 
 def root_user_home_bind_mount() -> Dict[str, str]:
     return {"host_path": "/tmp", "container_path": "/root"}
+
+
+def has_at_least_one_checkpoint(experiment_id: int) -> bool:
+    # Loop through the trials to look for checkpoints
+    for trial in experiment_trials(experiment_id):
+        # Return true if at least one workload has a checkpoint
+        if len(workloads_with_checkpoint(trial.workloads)) > 0:
+            return True
+    # No checkpoints found for the experiment return false.
```

Comment:
please just remove these comments. in my opinion, they make the code harder to read.

Reply:
or maybe said better, a useful comment explains why something is done, especially something unusual or subtle, not just the mechanics of the code.

Reply:
Removed the comments.